packages/fluentui/react-northstar/src/components/Menu/MenuItem.tsx

```diff
@@ -215,6 +229,12 @@ export const MenuItem = compose<'a', MenuItemProps, MenuItemStylesProps, {}, {}>
 
     const slotProps = composeOptions.resolveSlotProps<MenuItemProps & MenuItemState>({
       ...props,
+      active,
```

Comment:
shouldn't we also resolve `accessibility` so that child behavior is applied correctly? If I use `tabBeavior` I would expect the items to be `tab`s, just like in shorthand

Reply:
Done, please check if it's the proper way :)